eagle-core/eagle-policy/eagle-policy-base/src/main/java/org/apache/eagle/policy/dao/PolicyDefinitionEntityDAOImpl.java

```diff
@@ -17,14 +17,17 @@
 package org.apache.eagle.policy.dao;
 
 import org.apache.eagle.alert.entity.AbstractPolicyDefinitionEntity;
+import org.apache.eagle.alert.entity.AlertDefinitionAPIEntity;
```

Comment:
Same as above, let us make PolicyDefinitionEntityDAO/PolicyDefinitionEntityDAOImpl not knowing specific subtype of alert definition. Ideally, all entity/daos that specific to Alerts should be refactor out from eagle-policies. We haven't achieve this goal, but let us keep it clear as much as possible